src/app/screens/BookDetail/layout.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+
+import BookAttr from './components/BookAttr/';
+import Suggestions from './components/Suggestions/';
+import NewComment from './components/NewComment/';
+import Comments from './components/Comments/';
+
+function BookDet({ book }) {
+  return (
+    <div>
+      {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
+      <Link to="/dashboard">
+        <h2 className="return">Volver</h2>
+      </Link>
+      <div className="detail-container">
+        <BookAttr book={book} />
+        <Suggestions book={book} />
+        <div>
+          <h1 className="comments-title">Comentarios</h1>
+          <NewComment book={book} />
+          <Comments book={book} />
+        </div>
+      </div>
+    </div>
+  );
+}
+
+export default BookDet;
```

Comment:
use a representative name

Reply:
maybe for all `index.js` you could add `Container` at the end, so the index could be `BookDetailContainer` and the layout `BookDetail`